tests/integration/relations/test_db_admin.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import asyncio
+import logging
+from pathlib import Path
+
+import pytest
+import yaml
+from pytest_operator.plugin import OpsTest
+
+from tests.integration.relations.helpers.helpers import (
+    get_userlist,
+    wait_for_relation_joined_between,
+)
+from tests.integration.relations.helpers.postgresql_helpers import (
+    check_database_creation,
+    check_database_users_existence,
+    get_unit_address,
+)
+
+FIRST_DISCOURSE_APP_NAME = "discourse-k8s"
+SECOND_DISCOURSE_APP_NAME = "discourse-charmers-discourse-k8s"
+REDIS_APP_NAME = "redis-k8s"
+APPLICATION_UNITS = 1
+DATABASE_UNITS = 3
+
+logger = logging.getLogger(__name__)
+
+METADATA = yaml.safe_load(Path("./metadata.yaml").read_text())
+PGB = METADATA["name"]
+PG = "postgresql-k8s"
+PSQL = "psql"
+APPS = [PG, PGB, PSQL]
```

Comment:
Are PSQL or APPS supposed to be used in some part of the test?

Reply:
nope, removed